src/mfs.jl

```diff
@@ -0,0 +1,170 @@
+
+
+
+function random_point_on_sphere(radius, n)
```

Comment:
There is a more efficient way to do this, that we use in StateSpaceSets.jl, in the sampler function: https://github.com/JuliaDynamics/StateSpaceSets.jl/blob/0f0e5224e3d7b2922bb6e408911ad932cbe22ef5/src/sampler.jl#L88-L103 You should utilize the statespace_sampler function directly so that we reduce the amount of total code written. You can re-initialize a random sampler via statespace_sampler each time the radius changes.

Reply:
I use this for testing: 
```
u = randn(n)
u = u./norm(u)
```
But `statespace_sampler` is better.